deploy/helm/elastic-agent/values.yaml

```diff
@@ -0,0 +1,455 @@
+      # @section -- 2 - Kubernetes integration
+      enabled: true
+      # -- jobs state stream vars
+      # @section -- 2 - Kubernetes integration
+      vars: {}
+  cronjobs:
+    state:
+      # -- enable cronjobs state stream (kube-state-metrics) [ref](https://www.elastic.co/docs/current/integrations/kubernetes/kube-state-metrics#state_cronjob)
+      # @section -- 2 - Kubernetes integration
+      enabled: true
+      # -- cronjobs state stream vars
+      # @section -- 2 - Kubernetes integration
+      vars: {}
+  persistentvolumes:
+    state:
+      # -- enable persistentvolumes state stream (kube-state-metrics) [ref](https://www.elastic.co/docs/current/integrations/kubernetes/kube-state-metrics#state_persistentvolume)
+      # @section -- 2 - Kubernetes integration
+      enabled: true
+      # -- persistentvolumes state stream vars
+      # @section -- 2 - Kubernetes integration
+      vars: {}
+  persistentvolumeclaims:
+    state:
+      # -- enable persistentvolumeclaims state stream (kube-state-metrics) [ref](https://www.elastic.co/docs/current/integrations/kubernetes/kube-state-metrics#state_persistentvolumeclaim)
+      # @section -- 2 - Kubernetes integration
+      enabled: true
+      # -- persistentvolumeclaims state stream vars
+      # @section -- 2 - Kubernetes integration
+      vars: {}
+  resourcequotas:
+    state:
+      # -- enable resourcequotas state stream (kube-state-metrics) [ref](https://www.elastic.co/docs/current/integrations/kubernetes/kube-state-metrics#state_resourcequota)
+      # @section -- 2 - Kubernetes integration
+      enabled: true
+      # -- resourcequotas state stream vars
+      # @section -- 2 - Kubernetes integration
+      vars: {}
+  services:
+    state:
+      # -- enable services state stream (kube-state-metrics) [ref](https://www.elastic.co/docs/current/integrations/kubernetes/kube-state-metrics#state_service)
+      # @section -- 2 - Kubernetes integration
+      enabled: true
+      # -- services state stream vars
+      # @section -- 2 - Kubernetes integration
+      vars: {}
+  system:
+    metrics:
+      # -- enable system metric stream (kubelet) [ref](https://www.elastic.co/docs/current/integrations/kubernetes/kubelet#system)
+      # @section -- 2 - Kubernetes integration
+      enabled: true
+      # -- system metric stream vars
+      # @section -- 2 - Kubernetes integration
+      vars: {}
+
+# -- extra [user-defined integrations](https://www.elastic.co/guide/en/fleet/current/elastic-agent-input-configuration.html) to be added to the Elastic Agent
+# An example can be found [here](./examples/nginx-custom-integration/README.md)
+# @section -- 3 - User Extra Integrations
+extraIntegrations: {}
+
+agent:
+  # -- enable elastic-agent
+  # @section -- 3 - Elastic-Agent Configuration
+  enabled: true
+  # -- elastic-agent version
+  # @section -- 3 - Elastic-Agent Configuration
+  version: 8.15.0
```

Comment:
I assume this needs to get bump every version. We should ensure this is added to the automation for this repo so that it gets bumped when we do new releases.

Reply:
@pkoutsovasilis are we intentionally not putting this under `appVersion` in Chart.yaml? Normally that indicates the Chart works with a wide array of application versions.

Reply:
yes I do that intentionally, because this Helm chart should be able to work with multiple elastic-agent versions. This was at least part of the old requirements, however maybe it is non-relevant any more?!